packages/kit/types/helper.d.ts

```diff
@@ -8,6 +8,17 @@ interface ReadOnlyFormData {
 	[Symbol.iterator](): Generator<[string, string], void>;
 }
 
+type ToJSON = { toJSON(...args: any[]): JSONValue };
+type JSONValue = Exclude<JSONString, ToJSON>;
+export type JSONString =
```

Comment:
No sure about the wording. Json string sounds like stringified json to me, but it's JSON itself. So that may be the better name, "JSON"

Reply:
I'm glad you commented on this, we've had a couple of discussion for this too on #2170 and ended up with `JSONResponse` then because it's used as a response, but since we'll want to reuse this somewhere else, "Response" doesn't quite cover it anymore.

"JSON" sounds perfect, but I'm not sure it it will collide with the built-in JSON object. I chose `JSONString` to indicate that anything defined here will be passed through `JSON.stringify`, but yeah it does sounds like the end result of stringified JSON.

Reply:
Mhm yeah that's true, we don't want the name clash. JSONObject?

Reply:
Object kind of restrict it since it can also be some primitives like string or number. `JSONable`? (Surprisingly, my spellchecker doesn't complain about this, which it does for `StringifiableJSON`)

Reply:
For me a JSON object is a JavaScript object which is valid JSON. Don't know if that's only me that has this semantic connection thoughö

Reply:
That would be `JSONValue` for me, which is already defined above. Naming is hard.